recipes/approxposterior/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "approxposterior" %}
+{% set version = "0.1" %}
+{% set sha256 = "83d9559b325ec45765e00b57992c9f6b8378387b45b9455ea22429b6bbce2d1c" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - pytest
+    - python
+    - pip
+    - numpy
+    - matplotlib
```

Comment:
Is this a build dep?

Reply:
Good catch! We pruned the builds deps to just include python.